java-tools/OsmAndServer/src/main/java/net/osmand/server/controllers/pub/WeatherController.java

```diff
@@ -50,6 +61,15 @@ public class WeatherController {
 	{
 		sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
 	}
+	
+	protected static final Map<String, Integer> SETTLEMENT_TYPES;
+	
+	static {
+		SETTLEMENT_TYPES = new HashMap<>();
```

Comment:
Use CityType structure, what this constant means? Approximate population could be part of CityType class